packages/google_sign_in/test/google_sign_in_test.dart

```diff
@@ -251,4 +253,40 @@ void main() {
       expect(await googleSignIn.signIn(), isNotNull);
     });
   });
+
+  group('GoogleSignIn with fake backend', () {
+    const MethodChannel channel = const MethodChannel(
```

Comment:
Don't define the channel here - just refer directly to `GoogleSignInPlugin.channel` below.